content/razor-views/reading/views-walkthrough/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Walkthrough: Wrapping up HelloASPDotNET"
+date: 2023-02-09T12:48:24-06:00
+draft: false
+weight: 6
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Welcome to the Walkthrough for creating and using templates in your `HelloASPDotNET` project.  
+
+
+You will find a video walkthrough of the steps covered in the reading.  Time stamps have been added to highlight the topics covered so far and their locations in the reading.
+
+The following topics are covered in the walkthrough:
+- creating a Razor template (View)
+- Home Views and `HomeController`
+- Shared Directory and `_Layout.cshtml` 
+- Refactor `Index` method to return a View instead of HTML
+- Create `Index` View
+- Refactor `Welcome` method to return a View instead of HTML
+- Create `Welcome` View
+- Use `Viewbag`
+- Use an conditional to make the view dynamic
+
+## Video Walkthrough
+{{< youtube ExCSx2x0mAM >}} 
```

Comment:
Update video link, and add chapters to the video.

Reply:
check check.